terraform/modules/aws/route53/main.tf

```diff
@@ -0,0 +1,32 @@
+
+# provider "aws" {
+#   region = "eu-central-1"
+# }
+
+resource "aws_route53_zone" "omega" {
+  name = var.zone_name
+}
+
+resource "aws_route53_record" "record" {
+
+  zone_id = aws_route53_zone.omega.zone_id
+  name    = var.record_name
+  type    = var.record_type
+  #ttl     = var.record_ttl
+  #records = var.record_records
+
+  #we don’t have alb, I specified entries in the alias of the current A record, then we will change it to substitute the values of alb.
+   alias {
+     name = "dualstack.alb-omega-2144136904.eu-west-1.elb.amazonaws.com."
```

Comment:
hmm seems this is a hardcoded name, can we use variables here to set it :thinking:

Reply:
Will update here as soon as the alb module is available. That’s why I immediately left a comment about why I did this now.

Reply:
yeah but the question here will you update it also using hardcode?

Reply:
No, we will not use hardcode. We will try to do it right, as the mentor taught)).

Reply:
@ggramal Can we skip this now if we fix it in the next PR, when there will be an ALB module?

Reply:
ok lets skip